dotnet/src/Connectors/Connectors.OpenAI/OpenAIServiceCollectionExtensions.cs

```diff
@@ -213,14 +214,125 @@ public static class OpenAIServiceCollectionExtensions
                 serviceProvider.GetService<ILoggerFactory>()));
     }
 
+    /// <summary>
+    /// Adds an Azure OpenAI text generation service with the specified configuration.
+    /// </summary>
+    /// <param name="builder">The <see cref="IKernelBuilder"/> instance to augment.</param>
+    /// <param name="config">Required configuration for Azure OpenAI.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="builder"/>.</returns>
+    public static IKernelBuilder AddAzureOpenAITextGeneration(
```

Comment:
I noticed that in some places you marked new methods as experimental, thanks for that! In scope of this work, we need to make every new class and method as experimental for initial period of time.

Reply:
Thanks for the hint. I addeed the missing attributes.